projects/novo-elements/src/elements/tooltip/Tooltip.directive.ts

```diff
@@ -40,10 +40,13 @@ export class TooltipDirective implements OnDestroy, OnInit {
   isHTML: boolean;
   @Input('tooltipCloseOnClick')
   closeOnClick: boolean = false;
+  @Input('tooltipOnOverflow')
+  overflow: boolean = false;
```

Comment:
Maybe to be consistent, we could call the variable: `onOverflow` just because the word `overflow` is quite overloaded.